spec/fixtures/contract_metered_usage_response.json

```diff
@@ -0,0 +1,9 @@
+{
+  "Id": "5e56923d9cc9ba14fc9c1e41",
+  "ContractId": "5e563b344de08409ace88ecb",
```

Comment:
This `ContractId` doesn't match the `stub_post` URL. Is this intentional?

Reply:
No it wasn't. Thanks